### PR TITLE
Get unread gmail details from your account

### PR DESCRIPTION
**Pull request by shyamkanani**

### CODE REVIEW

**Comment by the-ethan-hunt** [comment]
Is this required?
We already have a command to open Gmail and similarly Gmail Extensions available?
I am curious to know the vitality of this function @shyamkanani

File: windows/benji.py [hunk 1]
```diff
@@ -84,6 +87,94 @@ def events(frame,put):
...
+			user_id =  'me'
+			label_id_one = 'INBOX'
+			label_id_two = 'UNREAD' 
+			
+			unread_msgs = GMAIL.users().messages().list(userId='me',labelIds=[label_id_one, label_id_two]).execute()
+
+			mssg_list = []
+			messages = []
+			if 'messages' in unread_msgs:	
+				mssg_list = unread_msgs['messages']
```
Comment (sara-02) [inline windows/benji.py]: use the dictionary's `get("messages")` function instead of `["messages"]`

File: windows/benji.py [hunk 2]
```diff
@@ -84,6 +87,94 @@ def events(frame,put):
...
+			
+			final_list = []
+			i = i + 1
+			
+			for mssg in mssg_list:
+				temp_dict = { }
+				m_id = mssg['id'] 
+				message = GMAIL.users().messages().get(userId=user_id, id=m_id).execute() 
+				payld = message['payload'] 
+				headr = payld['headers']
```
Comment (sara-02) [inline windows/benji.py]: Use the `get()` function here as well.

**Comment by shyamkanani** [comment]
I think it's required @the-ethan-hunt.  No gmail extensions are not available. But instead of read one by one mail you can get details of all the unread mails in single window using this command.

**Comment by shyamkanani** [comment]
@sara-02, @the-ethan-hunt  any other changes required?

**Comment by sara-02** [comment]
@shyamkanani you need to convenience @the-ethan-hunt about the validity of the feature. Can you give a comparison of the most popular Gmail plugin for desktop and how is your's different? I agree with @the-ethan-hunt that the desktop plugins for email come with more capabilities like composing/deleting/archiving. Plus they can be configured to work with other email providers as well, and not be limited to Gmail.

**Pull request closed without merging**